extension/src/status.ts

```diff
@@ -91,15 +91,8 @@ export class Status extends Disposable {
       return
     }
 
-    if (this.available) {
-      return {
-        command: RegisteredCommands.EXTENSION_SETUP_WORKSPACE,
-        title: Title.SETUP_WORKSPACE
-      }
-    }
-
     return {
-      command: RegisteredCommands.SETUP_SHOW,
+      command: RegisteredCommands.SETUP_SHOW_DVC,
```

Comment:
Is this supposed to go to the setup page in general or the DVC Section?

Reply:
How about `SETUP_SHOW_DVC` if uninitialized otherwise `SETUP_SHOW`?

Reply:
Done!